server/src/main/java/org/opensearch/cluster/routing/allocation/decider/DiskThresholdDecider.java

```diff
@@ -581,13 +594,15 @@ private DiskUsageWithRelocations getDiskUsage(
         RoutingNode node,
         RoutingAllocation allocation,
         final Map<String, DiskUsage> usages,
+        final long avgFreeBytes,
+        final long avgTotalBytes,
         boolean subtractLeavingShards
     ) {
         DiskUsage usage = usages.get(node.nodeId());
         if (usage == null) {
             // If there is no usage, and we have other nodes in the cluster,
             // use the average usage for all nodes as the usage for this node
-            usage = averageUsage(node, usages);
```

Comment:
Do we still need the method `averageUsage` ?

Reply:
This was just called in a unit test testing just ```averageUsage``` function. Removed this along with unit test.